src/rpdk/core/contract/resource_client.py

```diff
@@ -354,10 +345,9 @@ def make_request(
                 "previousResourceProperties": previous_resource_state,
             },
             "region": region,
-            "awsPartition": partition,
             "awsAccountId": account,
             "action": action,
-            "callbackContext": callback_context,
+            "requestContext": {"callbackContext": callback_context},
```

Comment:
good catch

Reply:
nevermind on this. callbackContext is right